networkx/algorithms/community/tests/test_asyn_lpa.py

```diff
@@ -87,3 +87,23 @@ def test_several_communities():
     communities = asyn_lpa.asyn_lpa_communities(test)
     result = {frozenset(c) for c in communities}
     assert_equal(result, ground_truth)
+
+def test_two_communities():
+    test = Graph()
+
+    # c1
+    c1_edges = [(0, 2), (0, 3), (0, 4), (0, 5), (1, 2), (1, 4), (1, 7), (2, 4), (2, 5),\
```

Comment:
The backslash to indicate a line break is unnecessary inside a list literal, it can be safely removed.

Also, these lines are longer than 79 characters, and the continuation line is indented too much; they do not satisfy the requirements of [PEP8][1]; you may want to run [flake8][2] on your code to ensure it satisfies the PEP8 coding style conventions.

[2]: http://flake8.pycqa.org/en/latest/
[1]: https://www.python.org/dev/peps/pep-0008/